benchmark/sirun/strip-unwanted-results.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+
+const IGNORE_STATS = [
+  'system.time'
+]
+
+const lines = fs
+  .readFileSync(path.join(__dirname, 'results.ndjson'))
+  .toString()
+  .trim()
+  .split('\n')
+
+if (lines.length === 1 && lines[0] === '') {
+  console.log('The file "results.ndjson" is empty! Aborting...') // eslint-disable-line no-console
+  process.exit(1)
+}
+
+const results = []
+
+for (const line of lines) {
+  const obj = JSON.parse(line)
+
+  for (const iteration of obj.iterations) {
+    for (const stat of IGNORE_STATS) {
+      if (stat in iteration) {
+        delete iteration[stat]
```

Comment:
Is there a way we can just directly get what we need instead of having to remove the iterations here?

I guess it ideally, we could tell sirun to ignore a specific stat type?

Reply:
afaik, there is no way to select the outputs that we want, I can't see a way in the documentation or sirun repo.

Reply:
@bengl do you have an idea for that? :)